lib/discourse_hub.rb

```diff
@@ -15,7 +15,12 @@ def self.discourse_version_check
   end
 
   def self.discover_enrollment_payload
-    { include_in_discourse_discover: SiteSetting.include_in_discourse_discover? }
+    {
+      include_in_discourse_discover: SiteSetting.include_in_discourse_discover?,
+      forum_url: Discourse.base_url,
+      forum_title: SiteSetting.title,
+      locale: I18n.locale,
```

Comment:
`I18n.locale` works in this case because Sidekiq jobs always run with the `default_locale`, but I suggest to switch this to `SiteSetting.default_locale` to make it obvious that this sends the `default_locale` and not the current locale of a user.